french/telephone/prof_musique.md

```diff
@@ -1 +1,3 @@
 Comme c'est le weekend, sa mere repond et me demande d'aller lui acheter du pain.
+
+[Returner](/french/feu-de-camp.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.